src/DDTrace/Integrations/Nette/NetteIntegration.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace DDTrace\Integrations\Nette;
+
+use DDTrace\Contracts\Scope;
+use DDTrace\GlobalTracer;
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanData;
+use DDTrace\Tag;
+use DDTrace\Type;
+use Nette\Application\Application;
+
+class NetteIntegration extends Integration
+{
+    const NAME = 'nette';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function requiresExplicitTraceAnalyticsEnabling()
+    {
+        return false;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return self::NOT_AVAILABLE;
+        }
+
+        $integration = $this;
+
+        \DDTrace\hook_method('Nette\Configurator', '__construct', function () use ($integration) {
+            $integration->load();
+        });
```

Comment:
I think because this is defer-loaded you should call `integration->load()` instead of doing it though `hook_method`. I think.

Reply:
Nice catch @morrisonlevi! 

@Anilm3 As Levi pointed out, when you add this to the deferred loader, you no longer need to use `DDTrace\hook_method`. When I made this change locally, I can see the Nette spans in the UI now.

<img width="860" alt="Screen Shot 2021-05-04 at 9 53 21 AM" src="https://user-images.githubusercontent.com/578780/117014557-e921c400-acbe-11eb-89c2-b65539f2237e.png">

Reply:
For some reason I wasn't getting the spans with or without this with the Nette test application, but I think I had something not loading properly, indeed after setting everything correctly and removing removing the `hook_method` everything seems to be working as expected. Thanks!

Reply:
Actually this had some unexpected consequences as the root span was always overridden to be a `nette.request`, trying a different approach.

Reply:
My guess is that you are _also_ adding it to something in IntegrationsLoader -- IIRC we do not do that if they are deferred loaded. Is that right @SammyK ?

Reply:
Correct, deferred-loaded integrations only need to be [added to the `IntegrationsLoader` for PHP 5](https://github.com/DataDog/dd-trace-php/blob/9e0699a/src/DDTrace/Integrations/IntegrationsLoader.php#L104-L120) because PHP 5 does not support deferred loading.

Reply:
Thanks, this was the cause of the problem I was facing, things are working as expected now and I modified the code to account for the lack of deferred loading in PHP 5.